src/transformers/models/bert/modeling_bert.py

```diff
@@ -1153,7 +1155,7 @@ def forward(
 class BertLMHeadModel(BertPreTrainedModel):
 
     _keys_to_ignore_on_load_unexpected = [r"pooler"]
-    _keys_to_ignore_on_load_missing = [r"position_ids", r"predictions.decoder.bias"]
+    _keys_to_ignore_on_load_missing = [r"position_ids", r"predictions.decoder.bias", r"cls.predictions.decoder.weight"]
```

Comment:
Let me review this slowly 🙏  and verify a few things. But do you think `r"predictions.decoder.bias"` is a mistake and should be `r"cls.predictions.decoder.bias"` ?

I am going to check myself anyway.

Reply:
Maybe. I'm not sure but the test does yell if we're hiding a valid key (we I don't try to yell when we have an unused `_keys_to_ignore`. Want me to try and add it to the test ?